android/app/src/main/kotlin/io/sentry/mobile/app/MainActivity.kt

```diff
@@ -1,11 +1,11 @@
-package io.sentrymobile.app
+package io.sentry.mobile.app
 
 import androidx.annotation.NonNull
 import io.flutter.embedding.android.FlutterActivity
 import io.flutter.embedding.engine.FlutterEngine
 import io.flutter.plugin.common.MethodChannel
 import android.os.AsyncTask
-import io.sentrymobile.app.NativeCrashJava
+import io.sentry.mobile.app.NativeCrashJava
 
 class MainActivity: FlutterActivity() {
     private val CHANNEL = "app.sentrymobile.io/nativeCrash"
```

Comment:
what's about renaming this too so we match the app's package?

https://github.com/getsentry/sentry-mobile/blob/967a3f1585c492ad79e4e6228e0db684d5168e77/ios/Runner/AppDelegate.swift#L14
https://github.com/getsentry/sentry-mobile/blob/967a3f1585c492ad79e4e6228e0db684d5168e77/android/app/src/main/kotlin/io/sentrymobile/app/MainActivity.kt#L11
https://github.com/getsentry/sentry-mobile/blob/4080ac59161687e616a3505d85a2c482a59c70b3/lib/screens/debug/sentry_flutter_screen.dart#L14